charts/logzio-logs-collector/README.md

```diff
@@ -0,0 +1,134 @@
+# logzio-logs-collector
+
+**In development**
+
+kubernetes logs collection agent for logz.io based on opentelemetry collector
+
+## Prerequisites
+
+- Kubernetes 1.24+
+- Helm 3.9+
+
+Below is the extended README.md including the full configuration table based on the provided `values.yaml`.
+
+* * *
+
+Logz.io Logs Collector for Kubernetes
+=====================================
+
+The `logzio-logs-collector` Helm chart deploys a Kubernetes logs collection agent designed to forward logs from your Kubernetes clusters to Logz.io. This solution leverages the OpenTelemetry Collector, offering a robust and flexible way to manage log data, ensuring that your logging infrastructure scales with your application needs.
+
+Features
+--------
+
+*   **Easy Integration with Logz.io**: Pre-configured to send logs to Logz.io, simplifying setup and integration.
+*   **Secure Secret Management**: Option to automatically manage secrets for seamless and secure authentication with Logz.io.
+
+Getting Started
+---------------
+
+### Add Logz.io Helm Repository
+
+Before installing the chart, add the Logz.io Helm repository:
+
+```
+helm repo add logzio https://helm.logz.io helm repo update
+```
+
+### Installation
+
+1.  **Create the Logz.io Secret**
+    
+    If not managing secrets externally, ensure the Logz.io secret with your shipping token and other relevant information is created.
+    
+2.  **Install the Chart**
+    
+    Install `logzio-logs-collector` from the Logz.io Helm repository, with logzio authentication values:
+
+    ```
+    helm install logzio-logs-collector -n monitoring \
+    --set enabled=true \
+    --set secrets.logzioLogsToken=<<token>> \
+    --set secrets.logzioRegion=<<region>> \
+    --set secrets.env_id=<<env_id>> \
+    --set secrets.logType=<<logType>> \
+    logzio/logzio-logs-collector
+    ```
+
+    Replace `logzio-logs-collector` with your release name.
+    
+### Uninstalling the Chart
+
+To uninstall/delete the `logzio-logs-collector` deployment:
+
+shellCopy code
+
+`helm delete logzio-logs-collector`
```

Comment:
As it is in the install command we should consider specifying the namespace.
`helm delete logzio-logs-collector -n monitoring`